---
layout: default
title: Code
category: code
---

I have written a lot of code for my research. You can see most of it on my [GitHub Page](https://github.com/dcgerard). But here is a list of some of my packages:

- [ldsep](https://cran.r-project.org/package=ldsep): Linkage Disequilibrium Shrinkage Estimation for Polyploids
- [hwep](https://cran.r-project.org/package=hwep): Hardy-Weinberg Equilibrium in Polyploids
- [updog](https://cran.r-project.org/package=updog): Flexible Genotyping for Polyploids
- [tensr](https://cran.r-project.org/package=tensr): Covariance Inference and Decompositions for Tensor Datasets
- [seqgendiff](https://cran.r-project.org/package=seqgendiff): RNA-Seq Generation/Modification for Simulation
- [hose](https://github.com/dcgerard/hose): Higher-Order Spectral Estimators
- [vicar](https://github.com/dcgerard/vicar): Various Ideas for Confounder Adjustment in Regression

Most of my work emphasizes extreme reproducibility, linking code with
data, and executing automatically via a Makefile.

- Gerard (2021a): <https://github.com/dcgerard/hwesims>
- Gerard (2021b): <https://github.com/dcgerard/ld_simulations>
- Gerard (2021c): <https://github.com/dcgerard/ldfast_sims>
- Gerard and Stephens (2021): <https://github.com/dcgerard/ruvb_sims>
- Gerard (2020): <https://github.com/dcgerard/reproduce_fasims>
- Gerard and Stephens (2020): <https://github.com/dcgerard/mouthwash_sims>
- Gerard and Ferrão (2020): <https://github.com/dcgerard/reproduce_prior_sims>
- Gerard et al. (2018): <https://github.com/dcgerard/reproduce_genotyping>
- Gerard and Hoff (2017): <https://github.com/dcgerard/reproduce_sure>

### References

- Gerard, David. 2020. "Data-Based RNA-Seq Simulations by Binomial Thinning." *BMC Bioinformatics* 21 (1): 206. <https://doi.org/10.1186/s12859-020-3450-9>.

- Gerard, David. 2021a. "Double Reduction Estimation and Equilibrium Tests in Natural Autopolyploid Populations." *bioRxiv*. <https://doi.org/10.1101/2021.09.24.461731>.

- Gerard, David. 2021b. "Pairwise Linkage Disequilibrium Estimation for Polyploids." *Molecular Ecology Resources* 21 (4): 1230–42. <https://doi.org/10.1111/1755-0998.13349>.

- Gerard, David. 2021c. "Scalable Bias-Corrected Linkage Disequilibrium Estimation Under Genotype Uncertainty." *Heredity* 127 (4): 357–62. <https://doi.org/10.1038/s41437-021-00462-5>.

- Gerard, David, and Luís Felipe Ventorim Ferrão. 2020. "Priors for Genotyping Polyploids." *Bioinformatics* 36 (6): 1795–1800. <https://doi.org/10.1093/bioinformatics/btz852>.

- Gerard, David, Luís Felipe Ventorim Ferrão, Antonio Augusto Franco Garcia, and Matthew Stephens. 2018. "Genotyping Polyploids from Messy Sequencing Data.: *Genetics* 210 (3): 789–807. <https://doi.org/10.1534/genetics.118.301468>.

- Gerard, David, and Peter Hoff. 2017. "Adaptive Higher-Order Spectral Estimators." *Electron. J. Statist.* 11 (2): 3703–37. <https://doi.org/10.1214/17-EJS1330>.

- Gerard, David, and Matthew Stephens. 2020. "Empirical Bayes shrinkage and false discovery rate estimation, allowing for unwanted variation" *Biostatistics* 21 (1): 15–32. <https://doi.org/10.1093/biostatistics/kxy029>.

- Gerard, David, and Matthew Stephens. 2021. "Unifying and Generalizing Methods for Removing Unwanted Variation Based on Negative Controls." *Statistica Sinica* 31 (3): 1145–66. <https://doi.org/10.5705/ss.202018.0345>.
