---
layout: default
title: Code
category: code
---

I have written a lot of code for my research. You can see most of it on my [GitHub Page](https://github.com/dcgerard). But here is a list of some of my R packages:

- [menbayes](https://github.com/dcgerard/menbayes): Tests for Segregation Distortion in Tetraploids
- [ldsep](https://cran.r-project.org/package=ldsep): Linkage Disequilibrium Shrinkage Estimation for Polyploids
- [hwep](https://cran.r-project.org/package=hwep): Hardy-Weinberg Equilibrium in Polyploids
- [updog](https://cran.r-project.org/package=updog): Flexible Genotyping for Polyploids
- [tensr](https://cran.r-project.org/package=tensr): Covariance Inference and Decompositions for Tensor Datasets
- [seqgendiff](https://cran.r-project.org/package=seqgendiff): RNA-Seq Generation/Modification for Simulation
- [hose](https://github.com/dcgerard/hose): Higher-Order Spectral Estimators
- [vicar](https://github.com/dcgerard/vicar): Various Ideas for Confounder Adjustment in Regression

Most of my work emphasizes extreme reproducibility, linking code with
data, and executing automatically via a Makefile.

- Gerard (2024): <https://github.com/dcgerard/mbanalysis>
- Gerard (2023a): <https://github.com/dcgerard/rmbayes_sims>
- Gerard (2023b): <https://github.com/dcgerard/hwesims>
- Gerard (2022)<https://github.com/dcgerard/hwesupp>
- Gerard (2021a): <https://github.com/dcgerard/ldfast_sims>
- Gerard (2021b): <https://github.com/dcgerard/ld_simulations>
- Gerard and Stephens (2021): <https://github.com/dcgerard/ruvb_sims>
- Gerard (2020): <https://github.com/dcgerard/reproduce_fasims>
- Gerard and Stephens (2020): <https://github.com/dcgerard/mouthwash_sims>
- Gerard and Ferr√£o (2020): <https://github.com/dcgerard/reproduce_prior_sims>
- Gerard et al. (2018): <https://github.com/dcgerard/reproduce_genotyping>
- Gerard and Hoff (2017): <https://github.com/dcgerard/reproduce_sure>

### References

- **Gerard, D.**, Thakkar, M., Ferr&atilde;o, L. F. V. (2024). Tests for segregation distortion in tetraploid F1 populations. *bioRxiv*. \\
  [boiRxiv:2024.02.07.579361](https://doi.org/10.1101/2024.02.07.579361)

- **Gerard, D.** (2023a). Bayesian tests for random mating in polyploids. *Molecular Ecology Resources* 23(8), p. 1812--1822. \\
  [doi:10.1111/1755-0998.13856](https://doi.org/10.1111/1755-0998.13856)
  [bioRxiv:2022.08.11.503635](https://doi.org/10.1101/2022.08.11.503635)

- **Gerard, D.** (2023b). Double reduction estimation and equilibrium tests in natural autopolyploid populations. *Biometrics* 79(3), p. 2143--2156.\\
  [doi:10.1111/biom.13722](https://doi.org/10.1111/biom.13722)
  [bioRxiv:2021.09.24.461731](https://doi.org/10.1101/2021.09.24.461731)

- **Gerard, D.** (2022). Comment on three papers about Hardy-Weinberg equilibrium tests in autopolyploids. *Frontiers in Genetics* 13, p. 1--6.\\
  [doi:10.3389/fgene.2022.1027209](https://doi.org/10.3389/fgene.2022.1027209)
  [HAL:hal-03754674](https://hal.archives-ouvertes.fr/hal-03754674)

- **Gerard, D.** (2021a). Scalable Bias-corrected Linkage
  Disequilibrium Estimation Under Genotype
  Uncertainty. *Heredity* 127(4), p. 357--362.\\  
  [doi:10.1038/s41437-021-00462-5](https://doi.org/10.1038/s41437-021-00462-5)
  [bioRxiv:2021.02.08.430270](https://doi.org/10.1101/2021.02.08.430270)

- **Gerard, D.** (2021b). Pairwise Linkage Disequilibrium Estimation for Polyploids. *Molecular Ecology Resources* 21(4), p. 1230--1242.\\
  [doi:10.1111/1755-0998.13349](https://doi.org/10.1111/1755-0998.13349)
  [bioRxiv:2020.08.03.234476](https://doi.org/10.1101/2020.08.03.234476)

- **Gerard, D.**, & Stephens, M. (2021). Unifying and
  Generalizing Methods for Removing Unwanted Variation Based on
  Negative Controls. *Statistica Sinica* 31(3), p. 1145--1166. \\
  [doi:10.5705/ss.202018.0345](https://doi.org/10.5705/ss.202018.0345)
  [arXiv:1705.08393](https://doi.org/10.48550/arXiv.1705.08393)

- **Gerard, D.** (2020). Data-based RNA-seq simulations by
  binomial thinning. *BMC Bioinformatics* 21(1), 206, p. 1--14.\\
  [doi:10.1186/s12859-020-3450-9](https://doi.org/10.1186/s12859-020-3450-9)
  [bioRxiv:758524](https://doi.org/10.1101/758524)

- **Gerard, D.**, & Stephens, M. (2020). Empirical Bayes
  Shrinkage and False Discovery Rate Estimation, Allowing For Unwanted
  Variation. *Biostatistics*, 21(1), p. 15--32.\\
  [doi:10.1093/biostatistics/kxy029](https://doi.org/10.1093/biostatistics/kxy029)
  [arXiv:1709.10066](https://doi.org/10.48550/arXiv.1709.10066)

- **Gerard, D.** & Ferr&atilde;o, L. F. V. (2020). Priors for
  Genotyping Polyploids. *Bioinformatics* 36(6), p. 1795--1800.\\
  [doi:10.1093/bioinformatics/btz852](https://doi.org/10.1093/bioinformatics/btz852)
  [bioRxiv:751784](https://doi.org/10.1101/751784)

- **Gerard, D.**, Ferr&atilde;o, L. F. V., Garcia, A. A. F., &
  Stephens, M. (2018). Genotyping Polyploids from Messy Sequencing
  Data. *Genetics*, 210(3), p. 789--807. \\
  [doi:10.1534/genetics.118.301468](https://doi.org/10.1534/genetics.118.301468)
  [bioRxiv:281550](https://doi.org/10.1101/281550)

- **Gerard, D.**, & Hoff, P. (2017). Adaptive higher-order
  spectral estimators. *Electronic Journal of Statistics*, 11(2),
  p. 3703--3737. \\
  [doi:10.1214/17-EJS1330](https://doi.org/10.1214/17-EJS1330)
  [arXiv:1505.02114](https://doi.org/10.48550/arXiv.1505.02114)




