---
layout: default
title: Curriculum Vitae
category: cv
---

<p> A PDF version of my CV may be found <a href="gerard_cv.pdf">here</a>. A BibTex file containing citations to my papers may be found <a href={{ site.baseurl }}/research/gerard_bib.bib>here</a>.</p>

<hr>

<h2> Education </h2>
<ul>
  <li> Ph.D. Statistics, University of Washington, June 2015.</li>
  <li> M.S. Statistics, The Ohio State University, June 2012.</li>
  <li> B.S. Mathematics, The Ohio State University, June 2010.</li>
  <li> B.S. Molecular Genetics, The Ohio State University, June 2010.</li>
</ul>

<h2> Articles </h2>
<ul>
    <li> <b>Gerard, D.</b>, Ferr&atilde;o, L. F. V., Garcia, A. A. F., &
  Stephens, M. (2018). Harnessing Empirical Bayes and Mendelian
  Segregation for Genotyping Autopolyploids from Messy Sequencing
  Data. <i>bioRxiv</i> <a href="https://doi.org/10.1101/281550">[Link
    to bioRxiv]</a></li>
    <br>
  <li> <b>Gerard, D.</b>, & Stephens, M. (2018). Empirical Bayes
  Shrinkage and False Discovery Rate Estimation, Allowing For Unwanted
  Variation. <i>To appear in
    Biostatistics</i>. <a href="https://doi.org/10.1093/biostatistics/kxy029">[Link
      to Biostatistics]</a>
    <a href="https://arxiv.org/abs/1709.10066">[Link to arXiv]}</a></li>
  <br>
  <li> <b>Gerard, D.</b>, & Stephens, M. (2017). Unifying and Generalizing Methods for Removing Unwanted Variation Based on Negative Controls. arXiv preprint arXiv:1705.08393. <a href="https://arxiv.org/abs/1705.08393">[Link to arXiv]</a> </li>
  <br>
  <li> <b>Gerard, D.</b>, & Hoff, P. (2017). Adaptive higher-order
  spectral estimators. <i>Electronic Journal
  of Statistics</i>, 11(2), 3703-3737. <a href="http://dx.doi.org/10.1214/17-EJS1330">[Link to EJS]</a><a href="http://arxiv.org/pdf/1505.02114.pdf">[Link
  to arXiv]</a> </li>
  <br>
  <li> <b>Gerard, D.</b>, & Hoff, P. (2016). A higher-order LQ decomposition for separable covariance models. <i>Linear Algebra and its Applications</i>, 505, 57-84. <a href="http://dx.doi.org/10.1016/j.laa.2016.04.033">[Link to LAA]</a><a href="http://arxiv.org/pdf/1410.1094.pdf">[Link to arXiv]</a> </li>
  <br>
  <li> <b>Gerard, D.</b>, & Hoff, P. (2015). Equivariant minimax dominators of the MLE in the array normal model. <i>Journal of Multivariate Analysis</i>, 137, 32-49. <a href="http://dx.doi.org/10.1016/j.jmva.2015.01.020">[Link to JMVA]</a> <a href="http://arxiv.org/pdf/1408.0424.pdf">[Link to arXiv]</a></li>
  <br>
  <li> Pollack, J. D., <b>Gerard, D.</b>, & Pearl, D. K. (2013). Uniquely Localized Intra-Molecular Amino Acid Concentrations at the Glycolytic Enzyme Catalytic/Active Centers of Archaea, Bacteria and Eukaryota are Associated with Their Proposed Temporal Appearances on Earth. <i>Origins of Life and Evolution of Biospheres</i>, 43(2), 161-187. <a href="http://dx.doi.org/10.1007/s11084-013-9331-8">[Link to Springer]</a></li>
  <br>
  <li> <b>Gerard, D.</b>, Gibbs, H. L., & Kubatko,
  L. (2011). Estimating hybridization in the presence of coalescence
  using phylogenetic intraspecific sampling. <i>BMC evolutionary
  biology</i>, 11(1),
  291. <a href="http://dx.doi.org/10.1186/1471-2148-11-291">[Link to
      BMC]</a></li>
</ul>

<h2> Research Experience </h2>
<ul>
  <li> Postdoctoral Scholar under Professor <a href="http://stephenslab.uchicago.edu/">Matthew Stephens</a>: 09/2015 - Present</li>
  <ul>
    <li> Developing empirical Bayes methods to account for hidden
      confounding in large scale gene expression studies.</li>
    <li> Efficiently implementing these methods in R and C++.</li>
    <li> Rigorously testing these methods using Monte Carlo methods.</li>
    <li> Leading/mentoring graduate students on their individual projects.</li>
    <li> Authoring reports and academic papers.</li>
  </ul>
  <li> Research Assistant Under Professor
    <a href="https://pdhoff.github.io/">Peter Hoff</a>: 09/2013 - 06/2015</li>
  <ul>
    <li> Studied optimal estimation of covariance matrices in Gaussian
      arrays using the notions of equivariance.</li>
    <li> Developed novel tensor decompositions, creating higher-order
      versions of well-known matrix decompositions.</li>
    <li> Developed singular value shrinkage mean estimators for
      tensor-variate data.</li>
  </ul>
  <li> Research Assistant Under Professor
    <a href="https://statistics.stanford.edu/people/joseph-s-verducci">Joe
      Verducci</a>: 09/2011 - 06/2012</li>
  <ul>
    <li> Studied monotone association between gene expression and drug
      resistance within unspecified subtypes of cancer.</li>
  </ul>
  <li> Undergraduate Research Fellow Under Professors
    <a href="http://www.stat.osu.edu/~lkubatko/">Laura Kubatko</a> and
    <a href="http://www.biosci.ohio-state.edu/~eeob/gibbs/">H. Lisle
      Gibbs</a>: 04/2009 - 06/2010</li>
  <ul>
    <li> Studied the hybridization of two subspecies of Missouri
      Rattlesnakes and two populations of Black Ratsnakes.</li>
    <li> Duties included sequencing work in the lab, computer programming
      using Perl, and data analysis with various programs.</li>
  </ul>
</ul>


<h2> Teaching Experience </h2>
<ul>
  <li> Lecturer of introductory statistics 09/2017 - 12/2017</li>
    <ul>
      <li> Developed my own lectures, homeworks, and exams.</li>
      <li> This course was intended to be a standard introduction to
	Statistics for non-majors plus some light mathematical theory. The
	content included introductions to probability, inference, and the R
	statistical computing language.</li>
    </ul>
  <li> Lecturer of introductory statistics 06/2013 - 09/2013</li>
  <ul>
    <li> Developed my own lesson plans and slides from which I lectured three times a week.</li>
    <li> Created material for two teaching assistants during their
      small-group sections.</li>
    <li> Course content included comparisons of good sampling designs
      versus bad sampling designs, conditions under which claims of
      causality may be made, confounding variables, good and bad measures
      for answering questions of interest, describing relationships
      between variables, and good and bad graphics to describe data.</li>
  </ul>
  <li> Graduate Teaching Assistant 09/2012 - 06/2013</li>
  <ul>
    <li> Led four one-hour recitation sections of introductory
      statistics every week.</li>
    <li> These were small group discussions that acted primarily as
      support for the main content taught during lecture.</li>
  </ul>
  <li> Lecturer of introductory statistics 09/2011 - 12/2011</li>
  <ul>
    <li> Lectured to large groups of students using a pre-defined course plan.</li>
    <li> Created my own slides to follow along the lesson plans.</li>
  </ul>
  <li> Phylogenetic Workshop Assistant 06/2011 - 08/2011</li>
  <ul>
    <li> Led a group of undergraduate researchers studying HIV. We
      studied the association between evolutionary distance (measured by
      branch distance along estimated trees) and clinical variables
      collected by an HIV treatment center in Belgium.</li>
  </ul>
  <li> Phylogenetic Workshop Assistant 06/2010 - 08/2010</li>
  <ul>
    <li> Led lab sessions on Unix and various phylogenetic program
      packages.</li>
    <li> Led a group of undergraduates researching the evolutionary
      origins of flightless birds.</li>
    <li> Individually mentored an undergraduate student on phylogenetic
      research and statistical analysis with R. We tried to incorporate
      protein structure into our gene tree estimates while studying the
      evolutionary relationships of HPV.</li>
  </ul>
  <li> Undergraduate Teaching Assistant: 09/2008 - 12/2008</li>
  <ul>
    <li> Taught four one-hour recitation sections of intermediate
      algebra every week.</li>
  </ul>
</ul>


<h2>Awards</h2>

<ul>
  <li> <a href="https://www.stat.washington.edu/news/awards/">Z.W. Birnbaum Award</a> for the best general exam of the 2013-2014 academic year.</li>
  <br>
  <li> Distinguished University Fellowship (September 2010 - June 2012).</li>
  <br>
  <li> Graduated Summa Cum Laude (June 2010).</li>
  <br>
  <li> Award for excellent research presentation at the BMPS Poster Forum (23 April 2010).</li>
  <br>
  <li> Medalist Scholarship (September 2006 - June 2010):
    <ul>
      <li> For academic excellence and essay writing skills.</li>
    </ul>
  </li>
  <li> President's Salute Nominee (April 2010):
    <ul>
      <li> For academic excellence among peers from the college of Arts and Sciences.
    </ul>
  </li>
</ul>

<h2>Coding</h2>
<ul>
  <li> R Packages: <a href="https://cran.r-project.org/web/packages/tensr/index.html">tensr</a>,
    <a href="https://github.com/dcgerard/hose">hose</a>,
    <a href="https://github.com/dcgerard/vicar">vicar</a>,
    <a href="https://github.com/dcgerard/seqgendiff">seqgendiff</a>,
    <a href="https://github.com/dcgerard/updog">updog</a>.
  </li>
  <br>
  <li> Advanced training: R, \LaTeX.</li>
  <br>
  <li> Intermediate training: C++, Shell.</li>
  <br>
  <li> Familiar: Python, Matlab/Octave, Mathematica.</li>
</ul>


<h2>Data Analysis</h2>
<ul>
  <li> Data Munging (tidyverse). </li>
  <br>
  <li> Data Visualization (ggplot2). </li>
  <br>
  <li> Reproducible Data Analysis (rmarkdown/make). </li>
  <br>
  <li> Version Control (Git/Github). </li>
  <br>
  <li> Continuous Integration (Travis CI/AppVeyor). </li>
  <br>
  <li> R package development.</li>
</ul>
